New-language-adaptation/German/deployment/deployment.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "742c5e9c-fd48-46ca-af23-931e2cdae617",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://developer.download.nvidia.com/compute/machine-learning/frameworks/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+    "\n",
+    "# German ASR Pipeline Deployment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25b4b112-ebdf-48d3-aaee-f5694f5e08ae",
+   "metadata": {},
+   "source": [
+    "In this notebook, we are going through the steps to deploy a German ASR pipeline into production.\n",
+    "\n",
+    "**Important note:** This notebook should be run from the host OS, where it can access the `docker` command. NVIDIA GPU driver, docker and Nvidia-docker should be pre-installed. For NVIDIA GPU driver, See instructions at https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html. For Nvidia-docker, see https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#step-1-install-a-container-engine.\n",
```

Comment:
```suggestion
    "**Important:** This tutorial should be run from the host OS, where it can access the `docker` command. The NVIDIA GPU driver, Docker, and nvidia-docker should be pre-installed. For the NVIDIA GPU driver, refer to the instructions [here](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html). For nvidia-docker, refer to the instructions [here](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#step-1-install-a-container-engine).\n",
```

Reply:
are we purposely not providing a link to the Docker install instructions?